synapse/axon.py

```diff
@@ -26,8 +26,10 @@
 
 megabyte = 1024000
 gigabyte = 1024000000
-
+terabyte = 1024000000000
 chunksize = megabyte * 10
+threedays = ((60 * 60) * 24) * 3
+axontag = 'class.synapse.axon.Axon'
```

Comment:
axontag defined in https://github.com/vertexproject/synapse/blob/master/synapse/axon.py#L99 and https://github.com/vertexproject/synapse/blob/master/synapse/axon.py#L271.

Megabyte and gigabyte are also defined twice.